octomap_server/src/OctomapServer.cpp

```diff
@@ -408,7 +416,7 @@ void OctomapServer::insertCloudCallback(const sensor_msgs::PointCloud2::ConstPtr
   insertScan(sensorToWorldTf.getOrigin(), pc_ground, pc_nonground);
 #ifdef STAMPED_OCTOMAP_SERVER
   if( m_time_thresh > 0 ) {
```

Comment:
Since this is mixed with time_thres and I like the epoch better, let's set the default m_time_thresh to zero.
Also, when you add the epoch pub, add one for time_thresh, so we can change it externally and not just in the config.